frontends/p4/simplifyDefUse.cpp

```diff
@@ -1335,6 +1335,11 @@ class FindUninitialized : public Inspector {
         if (basetype->is<IR::Type_Stack>()) {
             if (expression->member.name == IR::Type_Stack::next ||
                 expression->member.name == IR::Type_Stack::last) {
+                auto save  = lhs;
```

Comment:
please add a comment explaining that accessing these fields implies reading the whole stack since these behave like a read-modify-write operation on the LHS.